docs/runbook/govnotify-templates.md

```diff
@@ -0,0 +1,90 @@
+# Gov.uk Notify Templates
+
+The app uses 6 different notification templates (3 SMS and 3 Email). You can find suggested copy for each notification below, listed by the environment variable we use in the app.
+
+We suggest creating seperate templates for staging and production so that you can test template changes without affecting the live app.
```

Comment:
```suggestion
We suggest creating separate templates for staging and production so that you can test template changes without affecting the live app.
```

Reply:
Ah good catch. That was copied from prod so there's a typo in there